samples/smartsyncexplorer/SmartSyncExplorer.html

```diff
@@ -176,23 +175,11 @@ <h1 class="title">Contacts</h1>
     },
 
     syncDown: function() {
-        var self = this;
-        var syncName = "smartSyncExplorerSyncDown";
-        cordova.require("com.salesforce.plugin.smartsync").getSyncStatus(syncName, function(sync) {
-            if (sync == null) {
-                // First time
-                var target = {type:"soql", query:"SELECT " + self.fieldlist.join(",") + " FROM Contact  LIMIT 10000"};
-                cordova.require("com.salesforce.plugin.smartsync").syncDown(target, "contacts", {mergeMode:Force.MERGE_MODE_DOWNLOAD.OVERWRITE}, syncName, self.handleSyncUpdate.bind(self));
-            }
-            else {
-                // Subsequent times
-                cordova.require("com.salesforce.plugin.smartsync").reSync(syncName, self.handleSyncUpdate.bind(self));
-            }
-        });
+        cordova.require("com.salesforce.plugin.smartsync").reSync("syncDownContacts" /* see usersyncs.json */, this.handleSyncUpdate.bind(this));
```

Comment:
We don't have a method for just running a sync by name (without defining it), so using reSync. reSync the first time == sync.